pkg/server/routes.go

```diff
@@ -40,7 +41,7 @@ func (srv *RegistrationServer) SetupRoutes() error {
 		unsecuredV1.GET("/health", healthCheckCtrl.GetHandler)
 		unsecuredV1.GET("/authconfig", authConfigCtrl.GetHandler)
 		unsecuredV1.GET("/segment-write-key", analyticsCtrl.GetDevSpacesSegmentWriteKey) //expose the devspaces segment key
-
+		//unsecuredV1.GET("/metrics", metricsCtrl.PrometheusHandler)
```

Comment:
is it still needed?

Reply:
removed in [631fdc6](https://github.com/codeready-toolchain/registration-service/pull/342/commits/631fdc67dfc3b7aadcf3233ef1ecff22d2455b14)